src/components/Room/RoomDetailOption/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { LandDetailResponse } from 'api/room/entity';
+import OPTION_CATEGORY from 'static/optionCategory';
+import cn from 'utils/ts/classnames';
+import styles from './RoomDetailOption.module.scss';
+
+interface OptionProps {
+  roomDetail: LandDetailResponse
+}
```

Comment:
option만 넣을 수 있고, 더 구체화할 수 있을 것 같습니다.
```suggestion
type OptionProps = {
  [key in typeof OPTION_CATEGORY[number]['img_code']]: boolean;
};
```

Reply:
roomDetail의 값이 boolean 값 외에도 여러 타입의 값들이 존재하는데 위의 방식으로 하면 타입 오류가 뜨지 않나요?

Reply:
roomDetail중 option값만 추출해서 props로 넘기면 됩니다.

Reply:
수정했습니다. [c07410d](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/39/commits/c07410d79bc8294aa5c477d278d914c359e28996)